jaxb-ri/HOWTO-build.txt

```diff
@@ -8,5 +8,10 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-Three words. "mvn clean install"
+JavaSE 11 and Maven 3.3.9 are required for the build.
```

Comment:
Build jobs are using latest maven - 3.6.0, should they be updated to conform to what is being said here? Or should the text contain 'or newer'?